cmd/captplanet/setup.go

```diff
@@ -66,8 +77,21 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 		if err != nil {
 			return err
 		}
-		identPath = filepath.Join(farmerPath, "ident")
-		_, err = peertls.NewTLSFileOptions(identPath, identPath, true, true)
+		farmerCA := provider.CASetupConfig{
+			CertPath: filepath.Join(farmerPath, "ca.cert"),
+			KeyPath:  filepath.Join(farmerPath, "ca.key"),
+		}
```

Comment:
looks like there isn't a way to specify difficulty and a bunch of other things.

here's my recommendation: just manually make these fields part of setupCfg directly:

```
	Difficulty  uint64 `help:"minimum difficulty for identity generation" default:"24"`
	Timeout     string `help:"timeout for CA generation; golang duration string (0 no timeout)" default:"5m"`
	Overwrite   bool   `help:"if true, existing CA certs AND keys will overwritten" default:"false"`
	Concurrency uint   `help:"number of concurrent workers for certificate authority generation" default:"4"`
```

then manually fill in all the setupconfigs using those values and basepath.

Reply:
or alternatively, make the paths not be part of the casetup struct, and instead be an argument